.Rbuildignore

```diff
@@ -44,3 +44,6 @@
 ^LICENSE\.md$
 ^docker\.sh$
 ^dry$
+^README\.Rmd$
+^pacta_data_qa\.Rmd$
+^project_settings\.yml$
```

Comment:
This change does not come from reverting commits.

Reply:
noted